modules/cnf-configuring-the-ptp-fast-event-publisher.adoc

```diff
@@ -32,10 +43,20 @@ spec:
     node-role.kubernetes.io/worker: ""
   ptpEventConfig:
     enableEventPublisher: true <1>
-    transportHost: amqp://<instance_name>.<namespace>.svc.cluster.local <2>
+    storageType: "example-storage-class" <2>
 ----
 <1> Set `enableEventPublisher` to `true` to enable PTP fast event notifications.
-<2> Set `transportHost` to the AMQ router that you configured where `<instance_name>` and `<namespace>` correspond to the AMQ Interconnect router instance name and namespace, for example, `amqp://amq-interconnect.amq-interconnect.svc.cluster.local`
+<2> Use the value you set for `storageType` to populate the `StorageClassName` field for the `PersistentVolumeClaim` (`PVC`) resource that the PTP Operator automatically deploys.
```

Comment:
```suggestion
<2> Use the value that you set for `storageType` to populate the `StorageClassName` field for the `PersistentVolumeClaim` (`PVC`) resource that the PTP Operator automatically deploys.
```

Reply:
@mburke5678 Can you PTAL and merge this typo fix please: https://github.com/openshift/openshift-docs/pull/58868